README.md

```diff
@@ -128,6 +130,24 @@ sql.query`select * from foo ${sql.join(arrayOfSqlInnerJoins, " ")}`;
 // select * from foo inner join bar on (bar.foo_id = foo.id) inner join baz on (baz.bar_id = bar.id)
 ```
 
+### `sql.raw(val)`
+
+Directly injects the value provided into the SQL statement. **WARNING:** This value
+has no sanitation whatsover. Do NOT use this with user data, as they will
+have direct access to the query itself.
+
+```js
+const direction = "ascending";
+
+sql.query`
+  select * from foo order by bar ${sql.raw(
+    direction === "ascending" ? "asc" : "desc"
```

Comment:
A _much_ better solution here would be:

```js
sql.query`select * from foo order by bar ${direction === 'ascending' ? sql.query`asc` : sql.query`desc`}`
```

`sql.raw` is bad and should be avoided at all costs. Composing `sql.query` is a much better solution.

Reply:
👍 Agreed, I strongly discourage the use of sql.raw.

Reply:
Huh, didn't realize that would work. Oversight on my part. Thanks for the insight!